internal/kafka/internal/handlers/cloud_providers_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	supportedProviders = config.ProviderConfig{
+	SupportedProviders = config.ProviderConfig{
```

Comment:
why is this made public? i.e why the change from small `s` to capital `S`?

Reply:
> why is this made public? i.e why the change from small `s` to capital `S`?

@machi1990 was mainly a mistake, I have reverted those changes, thanks!